terraform/modules/apps/frontend/main.tf

```
@@ -7,79 +7,66 @@ terraform {
   }
 }
 
-resource "aws_ecs_task_definition" "service" {
-  family                   = var.service_name
-  requires_compatibilities = ["FARGATE"]
-  container_definitions    = file("${path.module}/frontend.json")
-  network_mode             = "awsvpc"
-  cpu                      = 512
-  memory                   = 1024
-  task_role_arn            = var.task_role_arn
-  execution_role_arn       = var.execution_role_arn
-
-  proxy_configuration {
-    type           = "APPMESH"
-    container_name = "envoy"
-
-    properties = {
-      AppPorts         = "80"
-      EgressIgnoredIPs = "169.254.170.2,169.254.169.254"
-      IgnoredUID       = "1337"
-      ProxyEgressPort  = 15001
-      ProxyIngressPort = 15000
-    }
-  }
+data "aws_secretsmanager_secret" "secret_key_base" {
+  name = "frontend_app-SECRET_KEY_BASE"
 }
 
-resource "aws_ecs_task_definition" "console_definition" {
-  family                   = var.service_name
-  requires_compatibilities = ["FARGATE"]
-  container_definitions    = file("${path.module}/frontend_console.json")
-  network_mode             = "awsvpc"
-  cpu                      = 512
-  memory                   = 1024
-  task_role_arn            = var.task_role_arn
-  execution_role_arn       = var.execution_role_arn
+module "app" {
+  source                           = "../../app"
+  cpu                              = "512"
+  memory                           = "1024"
```

These can be numbers rather than strings.
```suggestion
  cpu                              = 512
  memory                           = 1024
```